src/common/session/session-entity.js

```diff
@@ -8,21 +8,29 @@ import { DEFAULT_EXPIRES_MS, DEFAULT_INACTIVE_MS, PREFIX } from './constants'
 import { LocalMemory } from '../storage/local-memory'
 import { InteractionTimer } from '../timer/interaction-timer'
 import { wrapEvents } from '../wrap'
-import { Configurable } from '../config/state/configurable'
+import { getModeledObject } from '../config/state/configurable'
 import { handle } from '../event-emitter/handle'
 import { SUPPORTABILITY_METRIC_CHANNEL } from '../../features/metrics/constants'
 import { FEATURE_NAMES } from '../../loaders/features/features'
 
+// this is what can be stored in local storage (not enforced but probably should be)
+// these values should sync between local storage and the parent class props
 const model = {
   value: '',
   inactiveAt: 0,
   expiresAt: 0,
   updatedAt: Date.now(),
-  sessionReplayActive: false,
+  sessionReplay: 0,
   sessionTraceActive: false,
```

Comment:
Should this also be a zero to match the data types we expect to get back from the rum call?

Reply:
This is probably a little confusing and could use some polishing, but this value is actually going to map to the "mode" of the session replay which (as of right now, could change) is the result of 2 local sample decisions

```
const MODE = {
  OFF: 0,
  FULL: 1,
  ERROR: 2
}
```